TomTom SDK Examples/Utils/UI Components/BackButton.swift

```diff
@@ -12,6 +12,7 @@ import SwiftUI
 struct BackButton: View {
     var width: CGFloat = 64
     var height: CGFloat = 64
+    var shadow: Bool = false
```

Comment:
better to use assertions for boolean variable names
```suggestion
    var useShadow: Bool = false
```

Reply:
Done!